lntest/itest/lnd_channel_backup_test.go

```diff
@@ -1016,6 +1076,30 @@ func testChanRestoreScenario(t *harnessTest, net *lntest.NetworkHarness,
 		t.Fatalf("On-chain balance not restored: %v", err)
 	}
 
+	// For our force close scenario we don't need the channel to be closed
+	// by Carol since it was already force closed before we started the
+	// recovery. All we need is for Carol to send us over the commit height
+	// so we can sweep the time locked output with the correct commit point.
+	if testCase.localForceClose {
```

Comment:
👍

Reply:
As you mentioned in the OP, in the future we could provide the brute force logic _locally_ in the chain watcher if we detect this case.